.github/workflows/build_and_test.yml

```diff
@@ -201,6 +209,7 @@ jobs:
     - run: yarn format -c
 
     - name: Run yarn test
+      if: matrix.arch == 'x64'
```

Comment:
Do we need this test? (Are you expecting to put combinations in the matrix that won't run?)

Reply:
I was first planning to add a matrix of multiple OSes (ubuntu-latest, windows-latest, macos-latest) and archs (x64, arm64, ia32), but then we can only run tests on x64 since GH Actions is x64-only. It might be good to at least have the Node build task on the different OSes and archs (it would cross-compile to arm64 and ia32 then). Is that OK for you? Then I'm happy to update the workflow accordingly 😊

Reply:
Let's at least add all three OSs Desktop supports today. Cross-compiling without testing adds a little value in that we could be doing something terribly broken on ia32 (less likely for arm64), but I don't think it's enough to be worth it. It's not like we're interfacing with C types when it comes to the Node bridge.